projects/v2/src/app/store/ui.state.ts

```diff
@@ -86,7 +92,8 @@ export class UIStateModel {
     reportOpen: false,
     debugLogOpen: false,
     bottomSheetOpen: false,
-    compareOpen: false
+    compareOpen: false,
+    bottomSheetDOIOpen: false
```

Comment:
What is the point of this, where are you using it? I only see SheetState.getDOI.... being used

Reply:
I need to remove that. I tried to do something😅